pkg/investigations/ccam/ccam.go

```diff
@@ -22,6 +22,10 @@ var ccamLimitedSupport = ocm.LimitedSupportReason{
 func Evaluate(cluster *cmv1.Cluster, bpError error, ocmClient ocm.Client, pdClient pagerduty.Client, alertType string) error {
 	logging.Info("Investigating possible missing cloud credentials...")
 
+	if isAccessProtectionError(bpError.Error()) {
```

Comment:
Hmm.. I'm not sure the check should be in this investigation. Do you think it would make more sense to move it to https://github.com/openshift/configuration-anomaly-detection/blob/24e690daf7ff8f891bf51d77028791c865976bde/cadctl/cmd/investigate/investigate.go#L200 next to the GCP check?

Reply:
I am thinking the same yes. We can query ocm for the access protection status instead of going through a backplane failure.

Reply:
Yup, let's do that! :)